aws/resource_aws_storagegateway_gateway.go

```diff
@@ -138,6 +139,16 @@ func resourceAwsStorageGatewayGateway() *schema.Resource {
 				Optional:     true,
 				ValidateFunc: validateArn,
 			},
+			"smb_security_strategy": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ValidateFunc: validation.StringInSlice([]string{
+					storagegateway.SMBSecurityStrategyClientSpecified,
+					storagegateway.SMBSecurityStrategyMandatoryEncryption,
+					storagegateway.SMBSecurityStrategyMandatorySigning,
+				}, false),
```

Comment:
Might be good to rebase this and flip it to `_Values()` 👍 

```suggestion
				ValidateFunc: validation.StringInSlice(storagegateway.SMBSecurityStrategy_Values(), false),
```